modules/rosa-edit-objects.adoc

```diff
@@ -85,11 +85,32 @@ $ rosa edit ingress --cluster=<cluster_name> | <cluster_id> [arguments]
 |--cluster
 |Required: The name or ID (string) of the cluster to which the ingress will be added.
 
+|--cluster-routes-hostname  
+|Components route hostname for OAuth, console, download.
```

Comment:
I noticed "and" isn't added where there are commas in this table. For example:
```suggestion
|Components route hostname for OAuth, console, and download.
```
Is there a reason for that?